classes/Cart.php

```diff
@@ -3786,11 +3786,23 @@ public function checkDiscountValidity($obj, $discounts, $order_total, $products,
     }
 
     /**
-     * Return useful information about the cart.
+     * Return useful information about the cart for display purpose.
      *
      * @return array Cart details
      */
     public function getSummaryDetails($id_lang = null, $refresh = false)
+    {
+        $summary = $this->getUnalteredSummaryDetails($id_lang, $refresh);
+
+        return $this->alterSummaryForDisplay($summary, $refresh);
+    }
+
+    /**
+     * Returns useful raw informations about the cart.
```

Comment:
Information is an uncountable noun = no plural form

```suggestion
     * Returns useful raw information about the cart.
```

Reply:
Maybe describe the difference a little more in comments?